docs/html/reference/pip_install.rst

```diff
@@ -641,17 +680,38 @@ against any requirement not only checks that hash but also activates a global
 .. _`--require-hashes`:
 
 Hash-checking mode can be forced on with the ``--require-hashes`` command-line
-option::
+option:
+
+.. tabs::
+
+   .. group-tab:: Unix/macOS
+
+      .. code-block:: console
+
+          $ python -m pip install --require-hashes -r requirements.txt
+              ...
+              Hashes are required in --require-hashes mode (implicitly on when a hash is
+              specified for any package). These requirements were missing hashes,
+              leaving them open to tampering. These are the hashes the downloaded
+              archives actually had. You can add lines like these to your requirements
+              files to prevent tampering.
+                  pyelasticsearch==1.0 --hash=sha256:44ddfb1225054d7d6b1d02e9338e7d4809be94edbe9929a2ec0807d38df993fa
+                  more-itertools==2.2 --hash=sha256:93e62e05c7ad3da1a233def6731e8285156701e3419a5fe279017c429ec67ce0
```

Comment:
```suggestion
         $ python -m pip install --require-hashes -r requirements.txt
         ...
         Hashes are required in --require-hashes mode (implicitly on when a hash is
         specified for any package). These requirements were missing hashes,
         leaving them open to tampering. These are the hashes the downloaded
         archives actually had. You can add lines like these to your requirements
         files to prevent tampering.
           pyelasticsearch==1.0 --hash=sha256:44ddfb1225054d7d6b1d02e9338e7d4809be94edbe9929a2ec0807d38df993fa
           more-itertools==2.2 --hash=sha256:93e62e05c7ad3da1a233def6731e8285156701e3419a5fe279017c429ec67ce0
```